src/ComponentCatalogPanel.js

```diff
@@ -25,15 +26,20 @@ class ComponentCatalogPanel extends Component {
   }
 
   render() {
-    const {designLink, description, demos, docsLink, hero, sourceLink, title} = this.props;
+    const {designLink, description, demos, docsLink, hero, sourceLink, title, config} = this.props;
+    let heroComponent;
+
+    if (config) {
```

Comment:
I think add a TODO here saying remove this when we're done with the catalog update